model-optimizer/mo/front_ng/pipeline.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+
+import logging as log
+from mo.front_ng.extractor import fe_user_data_repack
+from mo.middle.passes.infer import validate_batch_in_shape
+
+
+def moc_pipeline(argv: argparse.Namespace):
+    from ngraph import Dimension, PartialShape        # pylint: disable=no-name-in-module,import-error
+    from ngraph.utils.types import get_element_type   # pylint: disable=no-name-in-module,import-error
+    log.info('New MOC pipeline')
+    fem = argv.feManager
+    log.info(f'fem.availableFrontEnds: {str(fem.get_available_front_ends())}')
+    log.info(f'Initializing new FE for framework {argv.framework}')
+    fe = fem.load_by_framework(argv.framework)
+    inputModel = fe.load_from_file(argv.input_model)
+
+    user_shapes, outputs, freeze_placeholder = fe_user_data_repack(
+        inputModel, argv.placeholder_shapes, argv.placeholder_data_types,
+        argv.output, argv.freeze_placeholder_with_value)
+
+    def compare_nodes(old, new):
+        eq = len(old) == len(new)
+        if eq:
+            for item in old:
+                found = [x for x in new if x['node'].is_equal(item)]
+                if not found:
+                    eq = False
+                    break
+        return eq
+
+    inputsEqual = True
+    if user_shapes:
+        inputsEqual = compare_nodes(inputModel.get_inputs(), user_shapes)
+
+    outputsEqual = True
+    if outputs:
+        outputsEqual = compare_nodes(inputModel.get_outputs(), outputs)
+    log.debug(f"Inputs are same: {inputsEqual}, outputs are same: {outputsEqual}")
```

Comment:
do not use `f"..."` approach. The some code for other places in the code

Reply:
Done. However 'pylint' recommends me to use this way

Reply:
@lazarevevgeny what is the reason? If it is only because python2 compatibility, so we already moved python version check to avoid main import.

Reply:
F-strings have been added in Python3.6, but we support Python3.5 as well.